examples/training_scripts/height_example/launch_height_benchmark_sagemaker.py

```diff
@@ -0,0 +1,79 @@
+# Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
This script should rather be in `examples/`, see my comments above.

Reply:
This is more or less a copy of examples/launch_height_benchmark_sagemaker.py but with an actual definition of the benchmark. I use it to launch remote experiments with sagemaker backend. I agree that the structure needs some polish since now we have two scripts that accomplish similar things which is suboptimal.